lib/y-richtext.coffee

```diff
@@ -215,10 +235,16 @@ class YRichText extends BaseClass
       throw new Error "This part of code must not be reached!"
 
   insertHelper : (position, content) ->
-    if content?
-      @_get("characters").insertContents position, content.split("") # convert content to an array
+    as_array =
+      if typeof content == "string"
```

Comment:
Thats not true. For example `4 == "4"` is true, but `4 === "4"` is not.. There are a lot of weird cases, and this is why you should use `==` only if you know what you are doing.